src/ArmTemplates/Common/API/Clients/Tags/TagClient.cs

```diff
@@ -58,20 +58,12 @@ public async Task<List<TagTemplateResource>> GetAllTagsLinkedToApiAsync(string a
 
         public async Task<List<TagTemplateResource>> GetAllTagsLinkedToProductAsync(string productName, ExtractorParameters extractorParameters)
```

Comment:
Can we add some tests for this?

Reply:
In my opinion there is no such logic that should be covered by tests (unit), it should be done before where the collection of the tags was done. Right now, it is just http request to retrieve data.

Reply:
Currently there are no E2E tests existing in resource-kit, Tom.
So I guess if we decide to add them at some moment - we will add the tests for all `<Entity>Client`s at once